src/iop/watermark.c

```diff
@@ -313,6 +319,17 @@ int default_colorspace(dt_iop_module_t *self, dt_dev_pixelpipe_t *pipe, dt_dev_p
   return iop_cs_rgb;
 }
 
+// sets text / color / font widgets sensitive based on watermark file type
+void text_color_font_set_sensitive(dt_iop_watermark_gui_data_t *g, gchar *filename)
+{
+  const gchar *extension = strrchr(filename, '.');
+  const gboolean active = !g_ascii_strcasecmp(extension, ".svg");
```

Comment:
we probably want to guard against extension being NULL.

Reply:
Well in theory we could do this only when adding files to the drop down list. After, all file should have extension.
Anyway I added the check everywhere, just in case